packages/extension/src/components/pages/SendPayment/PreparePayment/PreparePayment.tsx

```diff
@@ -126,14 +149,18 @@ export const PreparePayment: FC<PreparePaymentProps> = ({ onSendPaymentClick })
 
   const handleAmountChange = useCallback(
     (e: FocusEvent<HTMLInputElement>) => {
```

Comment:
@FlorianBouron FYI I have improved the amount field's UX a bit

Reply:
Thanks :+1 really appreciate it!